src/App.tsx

```diff
@@ -1,61 +1,143 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+}
+
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const users = [...usersFromServer];
+
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
```

Comment:
Use `null` for the default user id value - still actual from the previous review.
```suggestion
  const [userId, setUserId] = useState(null);
```

using `null` in such cases - is a good practice.